assets/csidriveroperators/aws-ebs/05_clusterrole.yaml

```diff
@@ -260,3 +260,10 @@ rules:
   - get
   - list
   - watch
+# Allow kube-rbac-proxy to create TokenReview to be able to authenticate Prometheus when collecting metrics
+- apiGroups:
+  - "authentication.k8s.io"
+  resources:.
+  - "tokenreviews"
+  verbs:
+  - "create"
```

Comment:
IIRC we also needed `subjectaccessreviews` for similar purpose. Is that still true or am I missing something?

Reply:
alright - lets see if that works. May be something special about going through a proxy.